DscResources/MSFT_xMySqlUser/MSFT_xMySqlUser.schema.mof

```diff
@@ -2,11 +2,11 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlUser")] 
 class MSFT_xMySqlUser : OMI_BaseResource
 { 
-  [Key, Description("Name of MySQL user to create or remove.")] String Name;
-  [Required, Description("Credential for MySql user."), EmbeddedInstance("MSFT_Credential")] String Credential;
-  [Required, Description("MySql connection credential used to create a user."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
-  [Write, Description("Ensure mysql user is present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-
+    [Key, Description("Name of MySQL user to create or remove.")] String UserName;
+    [Required, Description("Credential for MySQL user."), EmbeddedInstance("MSFT_Credential")] String UserCredential;
+    [Required, Description("MySQL root credential used to create a user."), EmbeddedInstance("MSFT_Credential")] String RootCredential;
+    [Write, Description("Ensure MySQL user is present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Required, Description("MySQL Version Number")] String MySqlVersion;
```

Comment:
Again I do not see any benefit of renaming the property Name to "UserName", Credential to "UserCredential" for the resource MySqlUser, only ConnectionCredential needs to be renamed to " RootCredential".

I also do not think the MySQL version should be  a required property for MySQLUser resource. What are the scenarios (if there even are any) where you would run multiple versions of MySQL server on the same machine and you need to create a user with the same name on both of the servers? At the time you create a user you already have MySQL installed, so the version can be programmatically calculated. I think the parameter mySqlVersion should be optional and default to my sql version found first on the machine during look up if the configuration script does not specify any version.